### ISSUE TITLE
GUI goes down after some hours no errors/ restart not possible

### ISSUE THREAD

**Issue by StijnDC** [SickRage/SickRage#3638]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: master / 1d28b358576b35b101bb0f662858ae5296d2ef8b
OS: Linux-4.4.27-12-osmc-armv6l-with-debian-8.0
What you did: nothing, RPI 1 Model B, running in the background
What happened: sickrage GUI unreachable after quite some time. force-reload/stop & start gives Git error.
What you expected: sickrage to stay up 24/7 or at least not crash. Have the ability to restart sickrage.

Mostly the crash happens after checking XEM scene exceptions. I've already turned off the custom URL, i had setup for extratorrent (extra.to) since it would crash every 20min.

Logs:
```
2017-04-18 18:47:02 INFO MAIN :: Writing PID: 600 to /home/****/.sickrage.pid
///Restart happens here.
2017-04-18 12:19:45 INFO SEARCHQUEUE-DAILY-SEARCH :: Checking for scene exception updates for AniDB
2017-04-18 12:19:41 INFO SEARCHQUEUE-DAILY-SEARCH :: Checking for XEM scene exception updates for theTVDB
2017-04-18 12:19:38 INFO SEARCHQUEUE-DAILY-SEARCH :: Checking for scene exception updates from http://sickrage.github.io/scene_exceptions/scene_exceptions.json
2017-04-18 12:19:38 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes
```
\\\LOG 2 after restart or force-reload
2017-04-17 18:03:48 INFO     CHECKVERSION :: Checking for updates using GIT
2017-04-17 18:03:50 INFO     CHECKVERSION :: Command git symbolic-ref -q HEAD didn't work
2017-04-17 18:03:51 ERROR    CHECKVERSION :: [1d28b35] Exception generated in thread CHECKVERSION: argument of type 'NoneType' is not iterable
Traceback (most recent call last):
  File "/home/****/.sickrage/sickbeard/scheduler.py", line 109, in run
    self.action.run(self.force)
  File "/home/****/.sickrage/sickbeard/versionChecker.py", line 67, in run
    if self.check_for_new_version(force):
  File "/home/****/.sickrage/sickbeard/versionChecker.py", line 269, in check_for_new_version
    if not self.updater.need_update():
  File "/home/****/.sickrage/sickbeard/versionChecker.py", line 585, in need_update
    if self.branch != self._find_installed_branch():
  File "/home/****/.sickrage/sickbeard/versionChecker.py", line 497, in _find_installed_branch
    branch_info, errors_, exit_status = self._run_git(self._git_path, 'symbolic-ref -q HEAD')  # @UnusedVariable
  File "/home/****/.sickrage/sickbeard/versionChecker.py", line 458, in _run_git
    if 'stash' in output:
TypeError: argument of type 'NoneType' is not iterable

**Comment by sharkykh** [comment]
Just to make sure, what is your git version?

**Comment by StijnDC** [comment]
it should be the latest version as i installed git one week ago.
Nevertheless, i think followed the wrong guide for a raspberry pi as I followed the guide for ubuntu.
 
I just got a new raspberry pi 3 and followed this guide for RPI OSMC:
https://discourse.osmc.tv/t/how-to-install-couchpotato-and-sickrage-on-raspberry-pi/10788

RPI has been up and running for +12hrs without issue.

I noticed the RPI1B was having CPU issues as wel, it was probably a python script that started hanging.
